overrides/kubejs/server_scripts/unify_barks.js

```diff
@@ -0,0 +1,40 @@
+ ServerEvents.tags('item', event => {
+//unify tags
+    event.add('forge:barks', '#immersive_weathering:bark')
+    event.add('forge:barks', '#farmersdelight:barks')
+ })
+ 
+
+ ServerEvents.recipes(event => {
+    event.replaceInput(
+        {input:'#farmersdelight:barks'},
+        '#farmersdelight:barks',
+        '#forge:barks')
+
+    event.replaceInput(
+        {input:'#immersive_weathering:bark'},
+        '#immersive_weathering:bark',
+        '#forge:barks')
+
+//Pale Jane needs overriding manually
+    event.remove({type: 'brewinandchewin:fermenting', output:'brewinandchewin:pale_jane'})

```

Comment:
it feels ever so slightly safe to make this go by ID instead, but ultimately it would make an unnoticeable difference.